scipy/stats/tests/test_resampling.py

```diff
@@ -1683,20 +1693,21 @@ def test_against_binomtest(self, alternative):
         # So we have to work with +1/-1 instead of 1/0.
 
         def statistic(x, axis=0):
-            return np.sum(x > 0, axis=axis)
+            xp_ = array_namespace(x)
+            return xp_.count_nonzero(x > 0, axis=axis)
 
         k, n, p = statistic(x), 10, 0.5
         expected = stats.binomtest(k, n, p, alternative=alternative)
 
-        res = stats.permutation_test((x,), statistic, vectorized=True,
-                                     permutation_type='samples',
-                                     n_resamples=np.inf, rng=self.rng,
-                                     alternative=alternative)
-        assert_allclose(res.pvalue, expected.pvalue, rtol=self.rtol)
+        res = stats.permutation_test((xp.asarray(x, dtype=xp.float64),), statistic,
```

Comment:
what happened to the `vectorized=True` argument? it looks like the test still passes with array API active if I leave that in; even if it has no effect, I wonder if we really want to be making simultaneous adjustment on top of the array API test mutations

Reply:
There's another case where it was removed below and it seemed to matter--it does add to reviewer overhead, especially without domain knowledge.

Reply:
I know. I try to keep it straightforward. Looks like I messed up this time.

Does adding at back address it?

Reply:
I do; see https://github.com/scipy/scipy/pull/23772#discussion_r2476443908.

Reply:
Added `vectorized=True` back here.